### PR TITLE
Add dockerfile

### LINKED ISSUE #49: Create Docker Container for web app
**I'm submitting a ...**  (check one with "x")
- [X] feature request

**Current behavior:**

I would like to see an automated build of the web app running in a docker container. I have a server so I could host the web app.
There is something similar, #3 but docker is also an option.

**Expected behavior:**

Please
- [ ] Create a docker file which runs the web app
- [ ] deploy this file to a web server (I can do this)

### PR DESCRIPTION
**Pull request by S2606**
Fixes #49 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+MAINTAINER Shagun Khemka "shagun.khemka60@gmail.com"
+RUN apt-get update -y
+RUN apt-get install -y python-pip python-dev build-essential
+COPY . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
```
Comment (niccokunzmann) [inline Dockerfile]: There is already a requirements.txt file in the app directory which would be /app/app/requirements.txt

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+click==6.7
```
Comment (niccokunzmann) [inline requirements.txt]: So, I think, you do not need to duplicate this.

**Comment by niccokunzmann** [comment]
@S2606 What do you think? Would you like to remove the duplication? I can also merge now and it can be done later.

**Comment by S2606** [comment]
@niccokunzmann I will remove the duplications

**Comment by S2606** [comment]
@niccokunzmann I have made changes

**Comment by jajodiaraghav** [comment]
Hi @S2606 Great work!

You might want to look at `git commit --amend` command, which basically merges the currently made changes to the previous commit.
Thanks

**Comment by S2606** [comment]
Hi @jajodiaraghav you mean `git commit --amend --no-edit` ?

**Comment by jajodiaraghav** [comment]
Yes. This (and other git commands) will greatly help you while making contributions.

**Comment by S2606** [comment]
@niccokunzmann @jajodiaraghav please review

**Comment by S2606** [comment]
You are welcome @niccokunzmann

**Pull request merged**